com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -1013,6 +1013,9 @@ internal void ShutdownInternal()
                 OnServerStopped?.Invoke(ConnectionManager.LocalClient.IsClient);
             }
 
+            // In the event shutdown is invoked within OnClientStopped or OnServerStopped, set it to false again
+            m_ShuttingDown = false;
```

Comment:
Why not just move the existing place where it's set to false, instead of setting it twice? Is there a reason it should always be false in OnServerStopped and OnClientStopped?

Reply:
If someone is checking ShutdownInProgress it would return true if it was not set to false... 
Yeah... it is a weird thing... so... OnServerStopped and OnClientStopped could invoke other (generic) methods that depend upon that value being false (i.e. validating that the shutdown has completed...which it really hasn't 100% completed...but we are invoking those callbacks at that point in time... so... one of those scenarios... keep it true and someone will complain that it still thinks it is shutting down when it is notifying that it stopped...set it to false...and someone might invoke shutdown again... for <insert various reasons here>... 😹 )

The compromise was to just make sure it was no longer shutting down after potentially invoking user code.

Reply:
That feels like it could use some conceptual rework... but not for 1.6. We can go with this for now...

Reply:
Yep... 💯